graphium/config/_loader.py

```diff
@@ -455,6 +455,11 @@ def load_accelerator(config: Union[omegaconf.DictConfig, Dict[str, Any]]) -> Tup
     merge_dicts(config, config_override)
     accelerator_type = get_accelerator(config_acc)
 
+    if accelerator_type == "gpu":
+        precision = config_acc.get("float32_matmul_precision", None)
+        if precision is not None:
```

Comment:
Why? I don't understand the logic? From my perspective you can always do the `set_float32_matmul_precision` since if the user uses float16 or float64, it will be ignored anyways

Reply:
My understanding of this line is that you can manually trade off between matmul-internal precision and speed (on some hardware). IMO this setting is user-specific, I wouldn't set it by default.
Source: https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision.